<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticketsystemClass</class>
 <widget class="QMainWindow" name="ticketsystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ticketsystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="text">
     <string>Total Number :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="text">
     <string>Start Selling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="text">
     <string>Current Number :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="title">
     <string>No.1 Window</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="title">
     <string>No.2 Window</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ticketsystem.qrc"/>
 </resources>
 <connections/>
</ui>
